CONTRIBUTING.md

```
@@ -53,23 +53,40 @@ Windows requires several additional tools that are not needed on macOS or Linux.
+Install the pinned toolchain and required components with `rustup`:
+
+```powershell
+rustup toolchain install 1.96.1 --profile minimal
+rustup component add rustfmt clippy --toolchain 1.96.1
+```
+
+**3. LLVM / Clang**
 
 `whisper-rs-sys` depends on `libclang`. Download the Windows x86_64 release from [github.com/llvm/llvm-project/releases](https://github.com/llvm/llvm-project/releases) (~822 MB). During install, check **"Add LLVM to system PATH for all users"**. If you see a "PATH too long" warning, skip the PATH step and set the environment variable manually:
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)